ssl/statem/statem_clnt.c

```diff
@@ -1183,7 +1183,6 @@ int tls_construct_client_hello(SSL *s, WPACKET *pkt)
 
     /* TLS extensions */
     if (!tls_construct_extensions(s, pkt, SSL_EXT_CLIENT_HELLO, NULL, 0, &al)) {
-        ssl3_send_alert(s, SSL3_AL_FATAL, al);
```

Comment:
Okay, thanks for the explanation.  (I guess I forgot that tls_construct_extensions() has lots of call sites.)